vmm/src/vm.rs

```diff
@@ -2152,6 +2152,25 @@ impl Vm {
         Ok(table)
     }
 
+    #[cfg(all(feature = "mshv", target_arch = "x86_64"))]
+    pub fn enable_memory_log_dirty(&self) -> std::result::Result<(), MigratableError> {
```

Comment:
This function can be merged to `Vm::start_memory_dirty_log()` by calling `hypervisor::Vm::enable_dirty_page_tracking()` directly with feature gate.